keps/prod-readiness/sig-apps/2599.yaml

```diff
@@ -2,4 +2,6 @@ kep-number: 2599
 alpha:
   approver: "@ehashman"
 beta:
-  approver: "@ehashman"  
+  approver: "@ehashman"
+stable:
+  approver: "@deads2k"
```

Comment:
David already has quite a lot PRRs this cycle - please switch to me and I will take a look soon.